src/custom/components/ClickWrap/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+import { ButtonPrimary, ButtonSecondary } from 'components/Button'
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 100%;
+  height: auto;
+  background: ${({ theme }) => theme.bg1};
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  min-height: 100px;
+  padding: 16px;
+  box-shadow: rgb(0 0 0) 0 -4px 0 0, rgb(0 0 0 / 16%) 0 -9px 0 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  font-size: 13px;
+  line-height: 1.2;
+
+  > div {
+    width: 100%;
+    max-width: 1000px;
+    align-items: center;
+    display: grid;
+    grid-column-gap: 24px;
+    grid-template-columns: 1fr 1fr;
+    justify-content: center;
+    justify-items: center;
+
+    ${({ theme }) => theme.mediaWidth.upToMedium`
+      max-width: 100%;
+      flex-flow: column wrap;
+      display: flex;
+    `};
+  }
+
+  > div > p {
+    text-align: left;
+    margin: 0;
+    width: 100%;
+
+    ${({ theme }) => theme.mediaWidth.upToMedium`
+      text-align: center;
```

Comment:
i think this should remain `text-align: left;` as above, with `margin: 0 auto`, and instead in the `mediaWidth` this becomes `width: some amount` so:
```css
> div > p {
    text-align: left;
    margin: 0 auto;
    width: 100%;
    ${({ theme }) => theme.mediaWidth.upToMedium`
      width: 150px; // <-- guessing on pxs here but u get the point
    ...
```

Reply:
Let's park this for now (click wrap will be disabled).